pkg/log/io/logger.go

```diff
@@ -0,0 +1,74 @@
+// Copyright 2023 The Okteto Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package io
+
+import (
+	"log/slog"
+
+	sloglogrus "github.com/samber/slog-logrus"
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	// JSONFormat represents a json logger
+	JSONFormat string = "json"
+)
+
+// oktetoLogger is the logger for the CLI information and debug logs
```

Comment:
what means information and debug logs? are they available when running a command? how can we get them?

Reply:
updated message